### PR TITLE
Added flutter widgets of the week links

### PR DESCRIPTION
**Pull request by JoseAlba**
#### Don't forget:
- [x] Are you sure you mean this repo? This repository is a reference to the [main Flutter repository](https://github.com/flutter/flutter). Code-level issues should be created there.
- [ ] Link to GitHub issues it solves. ```closes #1234```
- [x] Sign the CLA bot. You can do this once the pull request is opened.

Be sure to read the [Flutter guide on contributing](https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md).

### Related issues
 - Type `#` and the issue number or `#` and select the from the inline drop down.

### CODE REVIEW

File: docs/components/checkboxes.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ Flutter's APIs support accessibility setting for large fonts, screen readers, an
 - [Class description](https://api.flutter.dev/flutter/material/Checkbox-class.html)
 - [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/checkbox.dart)
 - [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/checkboxes/regular&gh_ref=develop)
+- [Youtube video](https://www.youtube.com/watch?v=RkSqPAn9szs&list=PLjxrf2q8roU23XGwz3Km7sQZFTdB996iG&index=82&t=35s&ab_channel=Flutter)
```
Comment (guidezpl) [inline docs/components/checkboxes.md:46]: Remove &t=35s

File: docs/components/buttons.md [hunk 2]
```diff
@@ -333,6 +333,7 @@ API and source code:
 - [Class definition](https://api.flutter.dev/flutter/material/ToggleButtons-class.html)
 - [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/toggle_buttons.dart)
 - [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/buttons/icon&gh_ref=develop)
+- [Youtube video](https://www.youtube.com/watch?v=kVEguaQWGAY&list=PLjxrf2q8roU23XGwz3Km7sQZFTdB996iG&index=63&ab_channel=Flutter)
```
Comment (guidezpl) [inline docs/components/buttons.md:336]: nit: YouTube instead of Youtube everywhere

**Review by guidezpl — APPROVED** [review]
```
  _    ___ _____ __  __ 
 | |  / __|_   _|  \/  |
 | |_| (_ | | | | |\/| |
 |____\___| |_| |_|  |_|
                        
```

File: docs/components/checkboxes.md (hunk 1 above)
Comment (JoseAlba) [reply docs/components/checkboxes.md]: DOne

File: docs/components/buttons.md (hunk 2 above)
Comment (JoseAlba) [reply docs/components/buttons.md]: Done

**Review by perclasson — APPROVED** [review]

File: docs/components/dividers.md [hunk 3]
```diff
@@ -60,7 +60,7 @@ Full-bleed dividers separate content into sections and span the entire length of
 - [Class definition](https://api.flutter.dev/flutter/material/Divider-class.html)
 - [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/divider.dart)
 - [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/dividers/full_bleed_divider&gh_ref=develop)
-
+- [YouTube video](https://www.youtube.com/watch?v=_liUC641Nmk&vl=en&ab_channel=Flutter)
```
Comment (rami-a) [inline docs/components/dividers.md:63]: New line after this

File: docs/components/progress_indicators.md [hunk 4]
```diff
@@ -80,7 +80,7 @@ Linear progress indicators display progress by animating an indicator along the
 - [Class definition](https://api.flutter.dev/flutter/material/LinearProgressIndicator-class.html)
 - [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/progress_indicator.dart)
 - [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/progress_indicators/linear&gh_ref=develop)
-
+- [YouTube video](https://www.youtube.com/watch?v=O-rhXZLtpv0&list=PLjxrf2q8roU23XGwz3Km7sQZFTdB996iG&index=75&ab_channel=Flutter)
```
Comment (rami-a) [inline docs/components/progress_indicators.md:83]: New line after this

File: docs/components/tabs.md [hunk 5]
```diff
@@ -119,6 +119,7 @@ Fixed tabs display all tabs on one screen, with each tab at a fixed width. The w
 - [Class definition](https://api.flutter.dev/flutter/material/TabBar-class.html)
 - [GitHub source](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/tabs.dart)
 - [Dartpad demo](https://dartpad.dev/embed-flutter.html?gh_owner=material-components&gh_repo=material-components-flutter&gh_path=docs/components/dartpad/tabs/fixed&gh_ref=develop)
+- [YouTube video](https://www.youtube.com/watch?v=POtoEH-5l40&list=PLjxrf2q8roU23XGwz3Km7sQZFTdB996iG&index=67&ab_channel=Flutter)
 - [Demo site](https://gallery.flutter.dev/#/demo/tabs)
```
Comment (rami-a) [inline docs/components/tabs.md:123]: Why does tabs have a link to the gallery but none of the other components do? We should be consistent

**Review by rami-a — APPROVED** [review]

**Comment by willlarche** [comment]
This is brilliant!

File: docs/components/dividers.md (hunk 3 above)
Comment (JoseAlba) [reply docs/components/dividers.md:63]: Done

File: docs/components/progress_indicators.md (hunk 4 above)
Comment (JoseAlba) [reply docs/components/progress_indicators.md:83]: Done

File: docs/components/tabs.md (hunk 5 above)
Comment (JoseAlba) [reply docs/components/tabs.md:123]: I agree

**Pull request merged**